cli/src/pcluster/utils.py

```diff
@@ -310,3 +310,10 @@ def load_json_dict(file_path):
         json_content = json.load(file)
 
     return json_content
+
+
+def get_attr(obj, attribute):
```

Comment:
Minor: I'd use `attributes` for input parameter to distinguish from for loop parameter